fn_cve_search/README.md

```diff
@@ -0,0 +1,139 @@
+#About This Package
```

Comment:
Is it possible to refactor these docs similarly to: https://github.com/neetinkandhare/resilient-community-apps/blob/alien-vault/fn_alienvault_otx/README.md

Reply:
yes